apis/python/test/common.py

```diff
@@ -7,6 +7,14 @@
 import tiledb
 from tiledb.vector_search.storage_formats import storage_formats, STORAGE_VERSION
 
+def get_test_data_path():
+    current_dir = os.path.dirname(__file__)
+    test_data_path = os.path.join(current_dir, "../../../test-data")
```

Comment:
Speaking of src/include/test/array_defs.h -- I don't know if there is an easy way to do this, but it would be useful to automatically derive the array names and array paths from a single location for both Python and C++ (i.e., automatically generate array_defs.h and the array names and paths used by Python from our spec).

Maybe an aspirational story.